packages/@vuepress/theme-default/components/Home.vue

```diff
@@ -78,6 +78,8 @@ export default {
   .hero
     text-align center
     img
+      object-fit: contain
```

Comment:
`object-fit` is unnecessary here since we didn't limit the size of the image container.